test/integration/rules/submit-button/submit-button.json

```diff
@@ -0,0 +1,7 @@
+{
+	"description": "submit-button test",
+	"rule": "submit-button",
+	"violations": [["#fail1"]],
+	"passes":  [["#pass1"], ["#pass2"], ["#pass3"]],
+	"incomplete": [["#undefiend1"]]
```

Comment:
Would like to see more test cases.

Reply:
Any test cases that comes to mind? Will add test cases for disabled buttons.